ENIGMAsystem/SHELL/Platforms/Win32/WINDOWScallback.cpp

```diff
@@ -60,6 +60,36 @@ namespace enigma
   static short hdeltadelta = 0, vdeltadelta = 0;
   static int tempLeft = 0, tempTop = 0, tempRight = 0, tempBottom = 0, tempWidth, tempHeight;
 
+  static WPARAM map_leftright_keys(WPARAM wParam, LPARAM lParam)
+  {
+    UINT scancode = (lParam & 0x00ff0000) >> 16;
+    int extended  = (lParam & 0x01000000) != 0;
```

Comment:
What are the constants for these fundies?

Reply:
nfi. https://docs.microsoft.com/en-us/windows/win32/inputdev/about-keyboard-input#keystroke-message-flags is where I found alt